src/Hosting/IntegrationTesting/src/Common/DeploymentParameters.cs

```diff
@@ -0,0 +1,157 @@
+// Copyright (c) .NET Foundation. All rights reserved.
+// Licensed under the Apache License, Version 2.0. See License.txt in the project root for license information.
+
+using System;
+using System.Collections.Generic;
+using System.IO;
+using System.Reflection;
+
+namespace Microsoft.Extensions.Hosting.IntegrationTesting
+{
+    /// <summary>
+    /// Parameters to control application deployment.
+    /// </summary>
+    public class DeploymentParameters
```

Comment:
Any reason why this is different from https://github.com/aspnet/AspNetCore/blob/18b81bacceacfd239359e6932742590e3f5094e3/src/Hosting/Server.IntegrationTesting/src/Common/DeploymentParameters.cs?